README.md

```diff
@@ -72,7 +73,13 @@ This employee also needs to pay taxes
            Tax Payable: £16.67
 
 Calculation rules:
- * Taxable income: Any amount of money earned above a gross annual salary of £11,000.00 is taxed at 20%
+ * Personal Allowance: Any amount of money earned up to £11,000.00.
```

Comment:
I want to introduce shared vocabulary for discussions with candidate. We don't define the name for 11,000 and this means that I use the name "Personal Allowance" and the other side use "Tax -free allowance", but they are not the same. In the context of a salary sleep "Tax-free allowance" is the monthly part of the personal allowance. In the context of the bands we can should speak about "Personal Allowance".